index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Build It Together</title>
```

Comment:
@thetechtakeover still is [missing the CSS file](https://www.internetingishard.com/html-and-css/hello-css/#linking-a-css-stylesheet). I also really liked the setup you had in the [HTML file you had](#12 ) for your last PR. It really helped set your fellow students up for success.